enterprise/api_client/lms.py

```diff
@@ -229,6 +243,20 @@ def get_course_enrollment(self, username, course_id):
 
         return result
 
+    def is_enrolled(self, username, course_run_id):
```

Comment:
There's a bug in this function - an enrollment can exist, but be inactive. This would appear to the user as if they were unenrolled, but the API will still return a record. If a record isn't returned, there's necessarily no enrollment, but if a record is returned, you'll need to look for the `is_active` key, as below:

```json
{
    "created":"2017-10-13T14:09:22.176782Z",
    "mode":"audit",
    "is_active":false,
    "course_details":{"course_id":"course-v1:edX+DemoX+Demo_Course","course_name":"edX Demonstration Course","enrollment_start":null,"enrollment_end":null,"course_start":"2013-02-05T05:00:00Z","course_end":null,"invite_only":false,"course_modes":[{"slug":"audit","name":"Audit","min_price":0,"suggested_prices":"","currency":"usd","expiration_datetime":null,"description":null,"sku":null,"bulk_sku":null}]},
    "user":"superegoe0df"
}
```

Hooray for manual testing ferreting this out!

Reply:
Ah, nice. I'm going to push and update the sandbox in a minute, when my local tests pass.

Reply:
This'll be on the sandbox in 5-10 minutes.

Reply:
It's up.